auth/credentials/detect.go

```diff
@@ -250,3 +253,31 @@ func clientCredConfigFromJSON(b []byte, opts *DetectOptions) *auth.Options3LO {
 		AuthStyle: auth.StyleInParams,
 	}
 }
+
+// GetGoogleMtlsTokenURL returns the mTLS token exchange URL
+func GetGoogleMtlsTokenURL() string {
```

Comment:
This feels like it should be a const and not a func if we want this public.

Reply:
Done